scripts/exgfs_wave_post_gridded_sbs.sh

```diff
@@ -81,18 +81,19 @@ cat << EOF
   INFO:     OUTPARS_WAV="${OUTPARS_WAV}"
 EOF
 
-if [[ "${DOGRB_WAV}" = "NO" ]]; then
+if [[ "${DOGRB_WAV}" == "NO" ]]; then
   echo "FATAL ERROR: DOGRB_WAV = NO"
   echo "FATAL ERROR: No grib2 products will be created, ABORT!"
-  exit 1
+  export err=1
+  err_chk
```

Comment:
Similar comment, move echos to `err_chk` message?

Reply:
Yes.  Fixed.